discover.py

```diff
@@ -93,6 +88,8 @@ def get_dns_conf_string(interface):
         'bind-interfaces',
         'interface={}'.format(interface),
         'listen-address={}'.format(ip),
+        'interface=lo',
+        'listen-address=127.0.0.1',
```

Comment:
Looks like `127.0.0.1` will show up twice in the config file, because you're also removing it from the blacklist below, right?